### PR TITLE
Edit screen header. No toolbar style.

### PR DESCRIPTION
**Pull request by kejmil01**

### CODE REVIEW

File: app/src/main/res/layout/activity_edit.xml [hunk 1]
```diff
@@ -1,14 +1,76 @@
...
+                        android:layout_height="match_parent"
+                        android:background="@drawable/edit_header_gradient" />
+
+                </FrameLayout>
+
+                <android.support.v7.widget.Toolbar
+                    android:id="@+id/main_toolbar"
+                    android:layout_width="match_parent"
+                    android:layout_height="?attr/actionBarSize"
+                    android:background="@null"
```
Comment (BartSoftware) [inline app/src/main/res/layout/activity_edit.xml]: Do we need this? Probably there will be some problems with proguard. Maybe just remove this background thing?

**Review by BartSoftware — APPROVED** [review]

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java [hunk 2]
```diff
@@ -23,18 +29,47 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
...
+        EditViewModel editViewModel = new EditViewModel(
+                imageDetails.getName(),
+                imageDetails.getPath(),
+                editCompoundRecyclerView,
+                new TagsProvider(imageDetails));
+        activityEditBinding.setViewModel(editViewModel);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java:74]: missing tests for this method

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+
 import com.invisibleteam.goinvisible.mvvm.images.adapter.ImagesCompoundRecyclerView;
 
-class ImagesViewModel {
+public class ImagesViewModel {
+
     private ImagesCompoundRecyclerView imagesCompoundRecyclerView;
     private ImagesProvider imagesProvider;
     private ImagesView imagesView;
+    private boolean isInitialized;
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java]: it's unused

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java [hunk 4]
```diff
@@ -21,5 +25,11 @@
     private void initRecyclerView() {
         imagesCompoundRecyclerView.updateResults(imagesProvider.getImagesList());
         imagesCompoundRecyclerView.setOnItemClickListener(imageDetails -> imagesView.navigateToEdit(imageDetails));
+        isInitialized = true;
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java]: it's unused

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java [hunk 5]
```diff
@@ -21,5 +25,11 @@
...
+    }
+
+    @VisibleForTesting
+    boolean isInitialized() {
```
Comment (michal1990) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/images/ImagesViewModel.java]: it's unused

**Pull request merged**